tests/publishing/templates/validations.test.js

```diff
@@ -7,44 +7,77 @@ function loginAndClickTemplates() {
       .then(() => common.clickSidebarTab(browser, 'Templates'))
 }
 
-//function somehowClearThePathField() { return Promise.resolve(); }
-
 describe('Template Validations', () => {
   before(() => loginAndClickTemplates());
+  after(() => common.logout());
+
+  it('should refuse a blank new template form', () => {
+    return common.waitAndClick('//button[@id="new_tmpl"]')
+      .then(() => common.waitAndClick('//*[@id="subBtn"]'))
+      .then(() => browser.waitForExist('//*[@id="admin-alert"]', 10000));
+  });
 
-  it('should refuse invalid template paths', () => {
-    let form = '//form[@id="newTmplForm"]';
-    return browser.click('//button[@id="new_tmpl"]')
-      .then(() => browser.setValue(`${form}//input[@id="path"]`, 'invalid input'))
-      .then(() => browser.click(`${form}//input[@name="name"]`))
+  it('should refuse an invalid template path in new template', () => {
+    return common.waitForOnClick('//button[@id="new_tmpl"]')
+      .then(() => browser.click('//button[@id="new_tmpl"]'))
+      .then(() => browser.setValue('//form[@id="newTmplForm"]//input[@id="path"]', 'invalid input'))
+      .then(() => browser.click('//form[@id="newTmplForm"]//input[@name="name"]'))
       .then(() => browser.getText('//span[@id="path-message"]'))
-      .then((text) => expect(text).to.equal("Path verification failed - can't mount destination."))
+      .then((text) => expect(text).to.equal("Path verification failed - can't mount destination."));
   });
 
-  // it('puts', () => {
-  //   expect(true).to.be.true;
-  // });
+  it('should refuse a new template form without a path', () => {
```

Comment:
Harping on the same logout after the above (that we expect to fail and make no change in the system), to then login and start this test that we also expect to make no changes in the system